test/time_tracker/test_helpers.clj

```diff
@@ -6,7 +6,14 @@
             [gniazdo.core :as ws]
             [clojure.test :as test]
             [clojure.spec.test :as stest]
-            [time-tracker.util :as util]))
+            [time-tracker.util :as util]
+            [clojure.string :as str]))
+
+(def test-port (Integer/parseInt (util/from-config :port)))
```

Comment:
Looking up config should be done at runtime. The form in `def` is evaluated at compile time, when the configuration may not necessarily be available.